services/services.rb

```diff
@@ -67,6 +67,15 @@ def self.user_service
       )
     end
 
+    # Start up a UserService from our JSONUserDataSource
+    def self.notification_service
+      @_notification_service ||= FastlaneCI::NotificationService.new(
+        notification_data_source: JSONNotificationDataSource.create(
+          File.expand_path("..", ci_config_git_repo_path)
```

Comment:
In the future, I think we should create like a `ci_data_home_path` which so we don't need to use `File.expand_path("..", ci_config_git_repo_path)`
but not now, just in the future :)